src/api/database/models/TranscriptNameMapping.ts

```diff
@@ -0,0 +1,34 @@
+import {
+    Table,
+    Column,
+    Model,
+    ForeignKey,
+    BelongsTo,
+    PrimaryKey,
+  } from 'sequelize-typescript';
+  import { UUID, STRING } from 'sequelize';
+  import User from './User';
+  import Transcript from "./Transcript";
+
+  @Table({ paranoid: true })
```

Comment:
we don't need paranoid mode it. (we should remove paranoid mode from most existing tables, too, in future prs).